sky/backends/cloud_vm_ray_backend.py

```diff
@@ -3440,9 +3440,10 @@ def cancel_jobs(self,
 
         Args:
             handle: The cluster handle.
-            jobs: Job IDs to cancel. If None, cancel all jobs.
-            cancel_latest_running_job: Whether to cancel the latest running job.
-                If set to True, asserts `jobs` is set to None.
+            jobs: Job IDs to cancel. If None or empty, cancel the latest running
+                job.
+            cancel_all: Whether to cancel all the jobs. If set to True, asserts
```

Comment:
Nit: I find it ambiguous to use "None or empty" to mean the same thing. It may be better to just use "None" to indicate the true absence of an arg.

Reply:
I feel it is fine to support both `None` and `empty` list, due to the [robustness principle](https://en.wikipedia.org/wiki/Robustness_principle). Or we can just keep the `core` to take both `None` and empty list but let the backend only take None. Wdyt?